org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeThingTypeMatcher.java

```diff
@@ -172,4 +172,30 @@ public RequiredProperty(String name, String value) {
             this.value = value;
         }
     }
+    
```

Comment:
It looks like this needs formatting as this method doesn't line up with the method above. It looks like tabs have been used rather than spaces?

Reply:
I accidentally applied spotless settings to all the code and had to cancel it later. This is probably a resulting error. I am going to fix it.

Reply:
I applied openhab formatter style to this method. It should be ok now. Same thing for the import order.